Le_duong/Database_01/View/viewAccount.php

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Management Accounts</title>
+	 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" 
+     integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
+<body>
+
+<?php 
+	include '../connect.php';
+	// Load data from db limit
+	if (isset($_GET['page']))
+	{
+		$page = (int)$_GET['page'];
+	}
+	else
+	{
+		$page = 1;
+	}
+	
+	$per_begin = 5;
+	$per_start = ($page - 1) * $per_begin;
+	$query = "SELECT * FROM accounts LIMIT $per_start,$per_begin";
+	$result = $conn->query($query);
+
+	// Handle sort 
+	if (isset($_GET["sort"]))
+	{
+		
+		$type = $_GET["sort"];
+		$ASC = "ASC";
+		$DESC = "DESC";
```

Comment:
```suggestion
		$asc = "ASC";
		$desc = "DESC";
```
tên biến viết bằng chữ thường